modules/veteran_verification/VETERAN_VERIFICATION.yml

```diff
@@ -256,42 +264,61 @@ components:
     ServiceHistoryEpisode:
       description: |
         Service History for authorized Veteran
+      required:
+        - id
+        - attributes
       type: object
       properties:
         id:
           type: string
+          nullable: false
           description: Service History Episode ID
           example: "12312AASDf"
         type:
           type: string
           example: service-history-episodes
         attributes:
           type: object
+          required:
+            - first_name
+            - last_name
+            - start_date
+            - end_date
+            - branch
```

Comment:
In PR comments, this is named `branch_of_service` but `branch` here. Is that intentional?